packages/peregrine/src/hooks/usePageTitle.js

```diff
@@ -0,0 +1,17 @@
+import { useCallback } from 'react';
+
+import { useDOMQuery } from './useDOMQuery';
+
+/**
+ * A hook that will return current page title and an updater.
+ */
+export const usePageTitle = () => {
+    const [, { setInnerText }] = useDOMQuery('title');
+    const updateTitle = useCallback(
+        newTitle => {
+            setInnerText(newTitle);
+        },
+        [setInnerText]
+    );
+    return updateTitle;
```

Comment:
In an effort to keep some consistency between hooks, can we have this return an array of the current value and an updater?

Ex: 
```js
const [elements, { setInnerText }] = useDOMQuery('title');
const titleElement = elements[0];

// other stuff

return [titleElement.innerText, updateTitle];
```

Reply:
@supernova-at Good suggestion.